src/PortingEmbed/PortingFormContainer.tsx

```diff
@@ -112,6 +143,17 @@ export function PortingFormContainer({
       setPortingUpdateError(
         e instanceof ApiError ? e.message : 'Something went wrong.'
       )
+      if (e instanceof ApiError) {
+        onError(e, {
+          code: e.code,
+          porting: subscription?.porting || undefined,
```

Comment:
optional: same thing :)
```suggestion
          porting: subscription?.porting,
```

Reply:
ah that's because porting can be undefined or null here